src/components/cutscenes/PokeCenterCutscene.ts

```diff
@@ -177,6 +178,10 @@ export class PokeCenterCutscene<TGameStartr extends FullScreenPokemon> extends C
                 this.gameStarter.scenePlayer.stopCutscene();
             });
         this.gameStarter.menuGrapher.setActiveMenu("GeneralText");
+        const map: string = this.gameStarter.itemsHolder.getItem("map");
```

Comment:
Put a blank line before the group of variable declarations to visually distinguish them from above.